Update ADHD dataset in examples to MAIN (#1953)
nilearn/nilearn: doc/connectivity/region_extraction.rst

.. currentmodule:: nilearn.datasets

Fetching movie-watching based functional datasets
=================================================

> We use naturalistic stimuli based movie watching functional connectivity datasets

Reviewer: ```suggestion
We use a naturalistic stimuli based movie-watching functional connectivity dataset
```

> of 20 subjects, which is already preprocessed and publicly available at

Reviewer: ```suggestion
of 20 subjects, which is already preprocessed, downsampled to 4mm isotropic resolution, and publicly available at
```

`<https://osf.io/5hju4/files/>`_. We use utilities

> :func:`fetch_development_fmri` implemented in nilearn for automatic fetching of these

Reviewer: ```suggestion
:func:`fetch_development_fmri` implemented in nilearn for automatic fetching of this
```

> datasets which is downsampled to 4mm.

Reviewer: Not sure why the 4mm downsampling is especially notable over other processing. Maybe we could add it above? Will try.
```suggestion
dataset.
```

Fetching resting state functional datasets
==========================================

We use resting state functional connectivity datasets of 20 subjects,
which is already preprocessed and publicly available at

> `<https://openneuro.org/datasets/ds000228/versions/1.0.0>`_. We use utilities

Reviewer: Should we actually give the OSF link here ? That's where the preprocessed data lives. The data available on OpenNeuro is either raw BIDS or smoothed.

Showing only one specific network regions before and after region extraction.

> Left image displays the regions of one specific movie-watching network without region extraction

Reviewer: ```suggestion
Left image displays the regions of one specific functional network without region extraction
```
